src/SQLite_Command.php

```diff
@@ -76,10 +76,14 @@ public function import( $args, $assoc_args ) {
 	 *  $ wp sqlite export --exclude_tables=wp_posts,wp_users
 	 *  Success: Exported to 'wordpress_dbase.sql'.
 	 *
-	 * @when before_wp_load
+	 * @when after_wp_load
```

Comment:
@jeroenpf, this change has been done, as in tests, `WP_SQLite_Translator` is instantiated as part of import and export. When the tests are running the `WP_SQLite_Translator`, try to call `do_action` which, in `before_wp_load` is undefined. That said, is it ok to call import and export after wp-load,  as above? Does this create a problem?
Here is part of the output:
```          thrown in /tmp/wp-cli-test-run--66ba10bc05cf53.68302961/wp-content/mu-plugins/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-translator.php on line 3992

        Fatal error: Uncaught Error: Call to undefined function do_action() in /tmp/wp-cli-test-run--66ba10bc05cf53.68302961/wp-content/mu-plugins/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-translator.php on line 3992

        Error: Call to undefined function do_action() in /tmp/wp-cli-test-run--66ba10bc05cf53.68302961/wp-content/mu-plugins/sqlite-database-integration/wp-includes/sqlite/class-wp-sqlite-translator.php```
Same goes with `import` as well.

Reply:
This will not work. If you try to import into a corrupt/empty/incomplete database, the command will fail if you load wp first. Same goes for export, it should work regardless of the state of the Wordpress installation.

I also think you might have an old version of the sqlite integration plugin because do_action and apply_filters instances are only executed conditionally if they exist. Can you double check with which version of the sqlite integration plugin you are testing?

Reply:
You might need to update wp-cli-tests repo. If you have an old version it will use a hardcoded version of sqlite integration plugin that is very outdated.

Reply:
Thanks for that @jeroenpf ! 
I manage to revert that by using the latest version.
Some notes:
1. The latest changes are not in a version, and I uses `dev-main` to make this work.
2. Updated the tests, as there were changes by using the latest sqlite plugin.
3. Reverted the `afte9r_wp_load` to `before_wp_load`.